lib/Storage/MySqlTimeSeriesStore.php

```diff
@@ -258,8 +271,11 @@ public function getStats($filterBy = [])
 
         $result = new TimeSeriesMySQLResults($statement);
 
-        return $result;
-
+        return
```

Comment:
Would it be better to add the `totalCount` as a property of the TimeSeriesMySQLResults class (and the Mongo one) rather than returning an array? It would make the interface concrete.

We could then have a getTotalCount() method on the TimeSeriesMySQLResults - don't forget to change the interface too.